### PR TITLE
[Nara] Remove channel and video deletion by moderators mappings and CLI commands

### PR DESCRIPTION
**Pull request by zeeshanakram3**
This PR addresses:

- #4740
- #4739
- Remove fixtures for above actions and fix integration tests

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1204404358291458/1205517269374078) by [Unito](https://www.unito.io)

### CODE REVIEW

**Review by mnaamani — CHANGES_REQUESTED** [review]
I don't think we should drop the mappings from the query-node because we need it to handle those events from previous runtimes, when we are processing blocks from genesis.

I left a comment about this in https://github.com/Joystream/joystream/pull/4874
Of course I also tried to fix the integration tests in that same PR and couldn't figure out why they were failing

https://github.com/Joystream/joystream/actions/runs/6218658040/job/16875580326?pr=4874#step:11:2141

Clearly you resolved it here :)

**Comment by mnaamani** [comment]
Also this needs to be rebased onto nara branch

**Comment by zeeshanakram3** [comment]
I have:

- Rebased the PR on `nara` branch
- removed the deleted events (`VideoDeletedByModerator` & `ChannelDeletedByModerator`) events from runtime
- Updated the chain metadata (there is a new metadata file `query-node/chain-metadata/2002.json`)
- Reverted the mappings deletion change.
- Updated the `@joystream/types` package based on new chain-metadata.
- I have added a _temporary_ tar dependency of the [hydra-typegen](https://github.com/Joystream/hydra/pull/531) fix to test the PR easily

File: query-node/codegen/package.json [hunk 1]
```diff
@@ -11,6 +11,6 @@
   },
   "dependencies": {
     "@joystream/hydra-cli": "5.0.0-alpha.4",
-    "@joystream/hydra-typegen": "5.0.0-alpha.4"
+    "@joystream/hydra-typegen": "file:../../joystream-hydra-typegen-v5.0.0-alpha.4.tgz"
```
Comment (mnaamani) [inline query-node/codegen/package.json]: We want to use the newly published [5.0.0-alpha.6](https://www.npmjs.com/package/@joystream/hydra-typegen/v/5.0.0-alpha.6)

**Review by mnaamani — CHANGES_REQUESTED** [review]

File: query-node/codegen/package.json (hunk 1 above)
Comment (zeeshanakram3) [reply query-node/codegen/package.json]: Yeah, sorry, figured out that I didn't save the file before adding the commit

**Review by mnaamani — APPROVED** [review]

**Comment by mnaamani** [comment]
Merged this work in https://github.com/Joystream/joystream/pull/4874

**Pull request merged**